scripts/ock-once-swtok.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+PIN=${OCK_PIN:-"12345678"}
+
+if id -u | grep -qw 0 && groups | grep -qwv pkcs11 ; then
+    echo "missing pkcs11 group membership"
+    exit 77
+fi
+
+if ! command -v pkcsslotd > /dev/null; then
+    echo "pkcsslotd is required"
+    exit 77
+fi
+
+for PREFIX in "/usr/local/" "/" "NA"; do
+    if [ "$PREFIX" = "NA" ]; then
+        echo "No ock configuration found or ock-swtok not configured"
```

Comment:
This stuff should be done in configure.ac, test should not even start if configure.ac has not found the needed stuff

Reply:
ok